#include <exception>

int main() {
    // 코드
    std::terminate();  // 비정상 종료
}